### PR TITLE
[CHORE] Adding website structure

### PR DESCRIPTION
**Pull request by nicolastakashi**
This PR I'm adding the website template as well as the automation scripts to build the website content.
I'll open a dedicated PR for the build and publish scripts

### CODE REVIEW

File: website/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    "preinfo": "npm version",
+    "info": "npm list",
+    "postinfo": "exec-bin node_modules/.bin/hugo/hugo version",
+    "postinstall": "shx rm -rf node_modules/.bin/hugo && shx mkdir node_modules/.bin/hugo && shx cp node_modules/gethyas/node_modules/.bin/hugo/* node_modules/.bin/hugo"
+  },
+  "dependencies": {
+    "@hyas/doks-core": "^1.1.1"
+  },
+  "engines": {
+    "node": ">=16.12.0",
```
Comment (Nexucis) [inline website/package.json:31]: shouldn't we use the v18 ?

File: website/netlify.toml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+[build]
```
Comment (Nexucis) [inline website/netlify.toml:1]: do you still want to use netlify to build and deploy the website ?

File: website/layouts/index.html [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        <h2 class="h4">Static Validation</h2>
+        <p>
+          Perses supports hassle-free dashboard-as-code usage with built-in static validation. Validate your dashboard
+          in CI/CD using the Perses CLI (percli).
+        </p>
+      </div>
+      <!-- <div class="col-lg-5">
+        <h2 class="h4">Read the docs</h2>
+        <p>Learn more in the <a href="https://getdoks.org/">Docs</a>.</p>
+      </div> -->
```
Comment (Nexucis) [inline website/layouts/index.html:53]: maybe you want to clean that section

**Review by Nexucis — COMMENTED** [review]
Overall I don't have much to say excepting thank you !

I still have some questions 👼 :

- regarding the script, why not implementing them in golang ? Could be easier no ?
- regarding the font in the vendor, isn't possible to download it with npm ?
- regarding the Perses logo, isn't possible to use the SVG instead of having multiple different size version of it ?

File: website/package.json (hunk 1 above)
Comment (nicolastakashi) [reply website/package.json:31]: I'd say yes, but I'll test 😄

**Comment by nicolastakashi** [comment]
> Overall I don't have much to say except thank you !
> 
> I still have some questions 👼 :
> 
> * regarding the script, why not implement them in golang? It could be easier no?
> * regarding the font in the vendor, isn't it possible to download it with npm?
> * regarding the Perses logo, isn't it possible to use the SVG instead of having multiple different size versions of it ?

My pleasure mate help always I can 😄 
Regarding your questions.
> * regarding the script, why not implement it in Golang? It could be easier no?
I'm a bit torn, I mean use go lang and execute shell commands from there I don't see many values, we could easily git client but I guess it will require more effort and I'm not sure about mdox if we could easily invoke it from golang without using any `cmd.run`.

The shell works fine, but I agree is not the easiest thing to read 😅 
If you're ok with the shell I'm too.

> * regarding the font in the vendor, isn't it possible to download it with npm?

I'm not sure, but I'll try
I'm currently using the docks template as it is, and in the mainstream, the fonts are available on the repo.

> * regarding the Perses logo, isn't it possible to use the SVG instead of having multiple different size versions of it ?

Yes, we can, we just need this one because I didn't find it on the repo, maybe I missed it.

File: website/netlify.toml (hunk 2 above)
Comment (nicolastakashi) [reply website/netlify.toml:1]: Not really, I'll delete it.

**Review by Nexucis — APPROVED** [review]
nice thanks !

**Pull request merged**